src/flashbar/collapsible.scss

```diff
@@ -208,6 +278,8 @@ the grid layout will be:
     flex-grow: 1;
     background: none;
     border: 0 none;
+    padding-top: 0;
```

Comment:
These two lines were moved from lines 140-141 because it only needs to affect `.button` and not `.status > .header` nor `.status > .item-count`. See #772 for more context on what these two lines are for.